.github/workflows/validate-pull-request.yml

```
@@ -9,10 +9,17 @@ on:
         description: >
           Configure as explained under https://github.com/marketplace/actions/jira-login#enviroment-variables. Note that
           it must NOT end with a slash.
+    inputs:
+      timeout-minutes:
+        required: false
+        type: number
+        default: 10
```

```suggestion
        default: 3
```
10 min is too much.
See: https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/actions/workflows/validate-pull-request.yml